iot/examples/README.md

```diff
@@ -17,7 +17,7 @@
   oc apply -f templates/build/enmasse-latest/install/components/example-plans
   
   oc apply -f templates/build/enmasse-latest/install/preview-bundles/iot
-  oc apply -f iot/examples/iot-config.yaml
+  oc apply -f templates/iot/examples/iot-config.yaml
```

Comment:
The patch should be "install/components/iot/examples/" for all these resources

Reply:
why was it moved there then ? 
https://github.com/EnMasseProject/enmasse/blob/master/templates/iot/examples/iot-config.yaml

Reply:
That's the source of templates ...